docs/index.bs

```diff
@@ -3195,6 +3238,11 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Else:
           1. Set |workerRealm| to the [=relevant realm=] of the |activeWorker|'s [=service worker/global object=].
           1. Set |eventHandled| to [=a new promise=] in |workerRealm|.
+          1. If |raceResponse| is not null, then:
+              1. Let |map| be |activeWorker|'s [=service worker/global object=]'s [=race response map=].
+              1. Let |token| be the result of [=generating a random UUID=].
+              1. [=map/Set=] |map|[|token|] to |raceResponse|.
+              1. Set |request|'s [=request/service-workers race token=] to |token|.
```

Comment:
Is it possible to just make the map keyed by requests, instead of strings? Then you don't need this token setup.

Reply:
Is it possible to get the map entry by using a struct (or object) as a key? If so that would be feasible.
On the other hand, the implementation side may become a bit difficult due to the type inconsistency of request.

Reply:
Yes. In specs maps can be keyed by anything. There's no need to change the implementation; as long as the spec is observably equivalent, it is OK for them to depart.

But, be sure to distinguish between requests and `Request`s in the spec...